tests/testutils.py

```diff
@@ -51,13 +95,14 @@ def setUp(self):
         )
 
         while not start_deferred.called:
-            # we need to advance until the pushkins have started up
+            # we need to wait for the database
             self.sygnal.reactor.advance(1)
             self.sygnal.reactor.wait_for_work(lambda: start_deferred.called)
```

Comment:
It looks like this is still waiting for `start_deferred`, which waits for the pushkins.